### DISCUSSION TITLE
I have this error fatal: refusing to merge unrelated histories

### DISCUSSION THREAD

**Discussion by Azurebluet98** [community/community#136781] [New to GitHub] [answered]
what can i do

**Comment by isma-is-dev** [comment, marked as answer]
Try this command
git pull origin master --allow-unrelated-histories

**Reply by Azurebluet98** [reply]
thanks

**Reply by ghostinhershell** [reply]
Thanks so much for lending a hand here @ismaelmartin302 🚀

**Reply by bkoronkowski** [reply]
I'm having this same issue and am new to Github.  Where exactly do I put this command in?  I need a step by step please.
